doc/env_vars.md

```diff
@@ -95,8 +95,8 @@ The following options are used for advanced database performance tuning and may
 
 | Env Var                         | Config                  | Description                                                                                                                                                                                                       | Default                                                                               |
 | ------------------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
-| `LRSQL_ADMIN_USER_DEFAULT`      | `adminUserDefault`      | The username of the account that seeds the account table, ie. added to the table upon initialization. Optional but **should be set in order to create other accounts**.                                           | Not set                                                                               |
-| `LRSQL_ADMIN_PASS_DEFAULT`      | `adminPassDefault`      | The password of the account that seeds the account table. Optional but **should be set in order to create other accounts**.                                                                                       | Not set                                                                               |
+| `LRSQL_ADMIN_USER_DEFAULT`      | `adminUserDefault`      | The username of the account that seeds the account table, ie. added to the table upon initialization. Optional but **should be set in order to create other accounts**. **NOTE: Not subject to validation.**      | Not set                                                                               |
```

Comment:
The phrasing "validation" is vague; I would change it to "Not subject to username length and character requirements." (Also you can remove the redundant `**` here.)

Reply:
3102d746bb0f2c7bd794c7b9c5a38e33b693e9f4